x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/dashboards/upgrade_risk_score.cy.ts

```diff
@@ -36,15 +36,14 @@ import { deleteRiskEngineConfiguration } from '../../../tasks/api_calls/risk_eng
 const spaceId = 'default';
 
 describe('Upgrade risk scores', { tags: ['@ess', '@serverless'] }, () => {
-  before(() => {
+  beforeEach(() => {
     login();
     deleteRiskEngineConfiguration();
```

Comment:
Not for now. But this test shouldn't delete anything in `beforeEach` hooks. All tests should delete what they create on `after/afterEach` hooks.